package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "node-openssl",
+  "version": "0.1.0",
+  "main": "main.js",
+  "private": true,
+  "gypfile": true,
+  "scripts": {
+    "start": "node-gyp configure build && node module.js",
+    "test": "node test.js"
```

Comment:
id have thought you'd use a testing lib to make sure your code meets some expectations, otherwise its jut ascript you run